src/mapper.rs

```diff
@@ -0,0 +1,91 @@
+use rustling_ontology_values::dimension::{Dimension,DatetimeKind};
+use rustling_ontology_values::output::OutputKind;
+pub use rustling_ontology_moment::{Interval, Moment, Local, TimeZone};
+
+pub fn map_dimension(dimension: &mut Dimension, output_kind_filter: &[OutputKind]) {
+
+    match dimension {
+
+        Dimension::Datetime(datetime_value) => {
+
+            // If the filter contains Datetime but no datetime subkind, then no subtyping is required
+            if output_kind_filter.contains(&OutputKind::Datetime) &&
+                !output_kind_filter.contains(&OutputKind::Date) &&
+                !output_kind_filter.contains(&OutputKind::DatePeriod) &&
+                !output_kind_filter.contains(&OutputKind::Time) &&
+                !output_kind_filter.contains(&OutputKind::TimePeriod) {}
```

Comment:
This has no effect. You must `return` if this condition is true.

Reply:
✅